logstash/types.db

```diff
@@ -0,0 +1,241 @@
+absolute		value:ABSOLUTE:0:U
+apache_bytes		value:DERIVE:0:U
+apache_connections	value:GAUGE:0:65535
+apache_idle_workers	value:GAUGE:0:65535
+apache_requests		value:DERIVE:0:U
```

Comment:
This file is taken from http://git.verplant.org/?p=collectd.git;a=blob;f=src/types.db;h=c605f56d603ce5d680a8f7b7180c27ab6e647f53;hb=b7de71b32b11645957e210ebadc86ca6cd84041f